products/sle/main.pm

```diff
@@ -789,6 +789,11 @@ sub load_consoletests() {
                 loadtest "console/zypper_lifecycle_toolchain";
             }
         }
+
+        if (sle_version_at_least('12-SP2')) {
+            loadtest "console/hwloc_testsuite";
+        }
+
```

Comment:
please make that a bit less verbose by removing the empty lines again and stating just

```perl
loadtest 'console/hwloc_testsuite' if sle_version_at_least('12-SP2');
```

Is this something that relates only to HPC? If yes, there is a block within the main.pm which is specific to HPC so it would probably fit better there.